### ISSUE TITLE
Continuing scikit-image related problem?

### ISSUE THREAD

**Issue by sixtyfive** [mittagessen/kraken#321]
After upgrading to 3.0.8 following your comment in #319:

```
$ cd ~/kraken-stable/
$ git pull
...
$ git branch
* (HEAD detached at 3.0.8)
  master
$ source env/bin/activate
(env) $ pip install . ; python setup.py install
...
Successfully installed kraken-3.0.8
...
(env) $ kraken --version
kraken, version 3.0.7 # assuming just the version bump missing?
(env) $ cd ~/testfiles
(env) $ kraken -d cuda:0 -I '*.png' -o '.xml' -a segment -bl -i ~/.../seg_best.mlmodel ocr -m ~/.../rec_best.mlmodel
WARNING:root:Torch version 1.10.2+cu102 has not been tested with coremltools. You may run into unexpected errors. Torch 1.9.1 is the most recent version that has been tested.
Loading ANN ~/.../seg_best.mlmodel     ✓
Loading ANN default     ✓
Segmenting      ✗
[18.0568] Failed processing vol03_page0045_f001.png: shape 
...
(env) $ pip list
Package             Version  
------------------- ---------
attrs               21.4.0   
certifi             2021.10.8
charset-normalizer  2.0.11   
click               8.0.3    
coremltools         5.1.0    
idna                3.3      
imageio             2.14.1   
importlib-metadata  4.10.1   
importlib-resources 5.4.0    
Jinja2              3.0.3    
jsonschema          4.4.0    
kraken              3.0.8    
lxml                4.7.1    
MarkupSafe          2.0.1    
mpmath              1.2.1    
networkx            2.6.3    
numpy               1.21.5   
packaging           21.3     
Pillow              9.0.0    
pip                 20.0.2   
pkg-resources       0.0.0    
protobuf            3.19.4   
pyparsing           3.0.7    
pyrsistent          0.18.1   
python-bidi         0.4.2    
PyWavelets          1.2.0    
regex               2022.1.18
requests            2.27.1   
scikit-image        0.19.1   
scipy               1.7.3    
setuptools          44.0.0   
Shapely             1.8.0    
six                 1.16.0   
sympy               1.9      
tifffile            2021.11.2
torch               1.10.2   
torchvision         0.11.3   
tqdm                4.62.3   
typing-extensions   4.0.1    
urllib3             1.26.8   
wheel               0.34.2   
zipp                3.7.0  
```

**Comment by mittagessen** [comment]
Arrrgh, I resolved the merge conflict in the cherry-picked commit incorrectly. It's fixed in 3.0.9.

**Comment by sixtyfive** [comment]
`master` still reports its version as `3.0.7`; not sure if that's intentional. bug is gone though, thanks!
